docs/BrowserObject.md

```diff
@@ -52,9 +52,9 @@ console.log(browser.requestedCapabilities)
  */
 ```
 
-## Get Config Options
+## Custom Configurations
```

Comment:
If changing the name is an issue due to breaking existing anchor links, I can revert this change. I think it's valuable, though, because `Config Options` just _adds_ to the confusion between Options and Configurations.

Reply:
I think this should be fine, can you find any references to this heading?

Reply:
Nope, not in the docs. I'd just be concerned about _external_ references to this anchor (e.g. bookmarks etc.)